zyte_common_items/util.py

```diff
@@ -48,38 +49,10 @@ def split_dict(dict: Dict, key_pred: Callable[[Any], Any]) -> Tuple[Dict, Dict]:
     return (no, yes)
 
 
-def url_to_str(url: Union[str, _Url]) -> str:
-    """Return the input :class:`~web_poet.RequestUrl` or
-    :class:`~web_poet.ResponseUrl` object as a string."""
-
-    if not isinstance(url, (str, _Url)):
-        raise ValueError(
-            f"{url!r} is neither a string nor an instance of RequestUrl or ResponseUrl."
-        )
-    return str(url)
-
-
-def format_datetime(dt) -> str:
-    """Return the specified :class:`~datetime.datetime` object, assumed to be
-    in the UTC timezone, in ISO format, with the timezone specified as
-    ``Z``."""
-    return f"{dt.replace(tzinfo=None).isoformat(timespec='seconds')}Z"
-
-
-def parse_iso_datetime(date_str) -> datetime.datetime:
```

Comment:
we don't need a backwards compat import for this, because it hasn't been released